test/Microsoft.DotNet.Cli.Utils.Tests/GivenAProjectDependenciesCommandFactory.cs

```diff
@@ -21,7 +21,7 @@ public void It_resolves_desktop_apps_defaulting_to_Debug_Configuration()
             var configuration = "Debug";
 
             var testAssetManager = new TestAssetsManager(Path.Combine(RepoRoot, "TestAssets", "DesktopTestProjects"));
-            var testInstance = testAssetManager.CreateTestInstance("AppWithDirectDependencyDesktopAndPortable")
+            var testInstance = testAssetManager.CreateTestInstance("AppWithDirectDepDesktopAndPortable")
```

Comment:
Why did we abbreviate Dependency to Dep? Ah, because of the path length on windows?